src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs

```diff
@@ -393,6 +393,16 @@ public void CopyTo(T[] array, int arrayIndex)
             Array.Copy(_items, 0, array, arrayIndex, _size);
         }
 
+        public void CopyTo(int sourceIndex, int count, Span<T> destination)
+        {
```

Comment:
The argument validation would be still needed here to prevent copying more than `List.Length` items and to use the right argument names in exception messages.

Reply:
Done